server/controllers/annotation_controller.js

```diff
@@ -54,10 +54,9 @@ export const getReplies = (user, parentId) => {
 };
 
 // PRECONDITION: user is not null.
-export const editAnnotation = (userId, annotationId, updateText) => {
-  const conditions = { _id: annotationId, authorId: userId };
```

Comment:
So should we be checking that the user wrote the annotation in a hook somewhere?